test/e2e/features/step-definitions/bookings-list.js

```diff
@@ -0,0 +1,40 @@
+import { defineSupportCode } from 'cucumber'
```

Comment:
This cucumber test is adding a booking for next week, and checking the week spinner by navigating to next week and seeing if that booking is displayed. There could be a concern that until cancelling a booking gets hooked up to the tests, this test could crowd the database with bookings, so I am open to suggestions/commenting this test out until cancelling gets integrated.

Reply:
for now, let's leave it and see what happens...